crcmocks/entitlements.py

```diff
@@ -1,28 +1,40 @@
 from flask import jsonify
 from flask import Blueprint
+from flask import request
+
+from crcmocks.config import DEFAULT_SERVICES
+from crcmocks.util import get_user_rh_identity
 
 
 blueprint = Blueprint("entitlements", __name__)
 
 
-services = [
-    "ansible",
-    "cost_management",
-    "insights",
-    "advisor",
-    "migrations",
-    "openshift",
-    "settings",
-    "smart_management",
-    "subscriptions",
-    "user_preferences",
-]
-entitlements = {}
-for svc in services:
-    entitlements[svc] = {"is_entitled": True, "is_trial": False}
-
-
-# TODO: allow customizable entitlements response per user
+# TODO: support is_trial?
 @blueprint.route("/v1/services")
 def services():
-    return jsonify(entitlements)
+    # get the identity header
+    identity_header = request.headers.get("X-Rh-Identity")
+
+    if not identity_header:
```

Comment:
Do we care if `identity_header` is only `None`? Then I think we should change this statement, to make it more obvious what may be returned. I'm assuming the `.get` is returning `None` of course :D

Reply:
Yeah for dealing with `None` I always go back and forth between:
```
if not <obj>
```
and 
```
if <obj> is None
```
The former is useful if you're checking strings/lists/dicts in addition to `None`, but here I think it's better to be explicit. Will update.

Reply:
Agreed, I do the same thing. Just makes the intent clearer that it being `None` *is* expected and isn't just silently getting _correctly_ grabbed from the conditional by chance.